plugin/server/services/updated-entry.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const { createCoreService } = require('@strapi/strapi').factories
+
+module.exports = createCoreService('plugin::translate.updated-entry', () => ({
+  async create(params) {
+    try {
+      const {
+        results: [firstResult],
+      } = await super.find({
+        fields: ['id'],
+        filters: {
+          contentType: params.data.contentType,
+          groupID: params.data.groupID,
+        },
+      })
+      if (firstResult) {
+        return super.update(firstResult.id, {
```

Comment:
I am always getting this error when updating an entity:

```
[2023-04-02 23:55:38.930] http: PUT /content-manager/collection-types/api::article.article/1014 (43 ms) 200
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
ValidationError: Invalid payload submitted for the update of an entity of type Translate updated Entry. Expected an object, but got undefined
    at Object.validateEntityUpdate (/home/felix/Documents/coding/strapi-develop/translate/node_modules/@strapi/strapi/lib/services/entity-validator/index.js:230:13)
    at Object.update (/home/felix/Documents/coding/strapi-develop/translate/node_modules/@strapi/strapi/lib/services/entity-service/index.js:184:45)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Object.<anonymous> (/home/felix/Documents/coding/strapi-develop/translate/node_modules/@strapi/strapi/lib/services/entity-service/index.js:327:20
```

Reply:
will look into this